.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
++#### Checklist
++
++- [ ] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
++- [ ] My branch is up-to-date with the Upstream `master` branch.
++- [ ] The tests pass locally with my changes.
++- [ ] I have added tests that prove my fix is effective or that my feature works(if appropriate).
++- [ ] I have added necessary documentation (if appropriate).
++
++#### Short description of what this resolves:
++
++
++#### Changes proposed in this pull request:
++
++-
++-
++-
++
++#### Fixes Issue
```

Comment:
The ISSUE_TEMPLATE.md uses `**asterisks**` for each section, but in this file we use `#### four hashes`. I would normalize and use the same one in both files (I don't have any preference *which*).